plugins/in_proc/in_proc.c

```diff
@@ -158,66 +158,29 @@ static pid_t get_pid_from_procname_linux(struct flb_in_proc_config *ctx,
 static int configure(struct flb_in_proc_config *ctx,
                      struct flb_input_instance *in)
 {
-    const char *pval = NULL;
-
-    /* interval settings */
-    pval = flb_input_get_property("interval_sec", in);
-    if (pval != NULL && atoi(pval) >= 0) {
-        ctx->interval_sec = atoi(pval);
-    }
-    else {
-        ctx->interval_sec = DEFAULT_INTERVAL_SEC;
-    }
+    char *proc_name = NULL;
+    int ret;
 
-    pval = flb_input_get_property("interval_nsec", in);
-    if (pval != NULL && atoi(pval) >= 0) {
-        ctx->interval_nsec = atoi(pval);
-    }
-    else {
-        ctx->interval_nsec = DEFAULT_INTERVAL_NSEC;
+    /* Load the config map */
+    ret = flb_input_config_map_set(in, (void *)ctx);
+    if (ret == -1) {
+        return -1;
```

Comment:
Could you output log ?
If `flb_input_config_map_set` failed, fluent-bit will exit without error logs.